fdbclient/SystemData.cpp

```diff
@@ -43,6 +46,10 @@ const KeyRangeRef keyServersKeyServersKeys(LiteralStringRef("\xff/keyServers/\xf
                                            LiteralStringRef("\xff/keyServers/\xff/keyServers0"));
 const KeyRef keyServersKeyServersKey = keyServersKeyServersKeys.begin;
 
+const UID anonymousShardId = UID(666666, 88888888);
```

Comment:
Could you add some comments on how these values are selected?

Reply:
They are selected randomly, and easy to find in logs.